yt_dlp/extractor/pokergo.py

```diff
@@ -0,0 +1,110 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import base64
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    try_get,
+)
+
+
+class PokerGoBaseIE(InfoExtractor):
+    _NETRC_MACHINE = 'pokergo'
+    _AUTH_TOKEN = None
+    _PROPERTY_ID = '1dfb3940-7d53-4980-b0b0-f28b369a000d'
+
+    def _login(self):
+        username, password = self._get_login_info()
+        if not username:
+            self.raise_login_required(method='password')
+
+        self.report_login()
+        self._AUTH_TOKEN = self._download_json(
+            f'https://subscription.pokergo.com/properties/{self._PROPERTY_ID}/sign-in', None,
+            headers={'authorization': f'Basic {base64.b64encode(f"{username}:{password}".encode()).decode()}'},
+            data=b'')['meta']['token']
+        if not self._AUTH_TOKEN:
+            raise ExtractorError('Unable to get Auth Token.', expected=True)
+
+    def _real_initialize(self):
+        if not self._AUTH_TOKEN:
+            self._login()
+
+
+class PokerGoIE(PokerGoBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?pokergo\.com/videos/(?P<id>[^&$#/?]+)'
+
+    _TESTS = [{
+        'url': 'https://www.pokergo.com/videos/2a70ec4e-4a80-414b-97ec-725d9b72a7dc',
+        'info_dict': {
+            'id': 'aVLOxDzY',
+            'ext': 'mp4',
+            'title': 'Poker After Dark | Season 12 (2020) | Cry Me a River | Episode 2',
+            'description': 'md5:c7a8c29556cbfb6eb3c0d5d622251b71',
+            'thumbnail': 'https://cdn.jwplayer.com/v2/media/aVLOxDzY/poster.jpg?width=720',
+            'timestamp': 1608085715,
+            'duration': 2700.12,
+            'season_number': 12,
+            'episode_number': 2,
+            'series': 'poker after dark',
+            'upload_date': '20201216',
+            'season': 'Season 12',
+            'episode': 'Episode 2'
+        },
+        'params': {'skip_download': True}
+    }]
+
+    def _real_extract(self, url):
+        id = self._match_id(url)
+        data_json = self._download_json(f'https://api.pokergo.com/v2/properties/{self._PROPERTY_ID}/videos/{id}', id,
+                                        headers={'authorization': f'Bearer {self._AUTH_TOKEN}'})['data']
+        v_id = data_json['source']
+
+        thumbnails = [{
+            'url': image['url'],
+            'id': image.get('label'),
+            'width': image.get('width'),
+            'height': image.get('height')
+        } for image in data_json.get('images') or [] if image.get('url')]
+        series_json = next(dct for dct in data_json.get('show_tags') or [] if dct.get('video_id') == id) or {}
+
+        return {
+            '_type': 'url_transparent',
+            'id': id,
```

Comment:
This should be display_id. Don't pass `id` to inner extractor if it will be different at the end